tests/integration/records/unload-test.js

```diff
@@ -130,44 +130,21 @@ test('can unload a single record', function(assert) {
         id: '1',
         attributes: {
           name: 'Adam Sunderland'
-        },
-        relationships: {
-          cars: {
-            data: [{
-              id: 1,
-              type: 'car'
-            }]
-          },
-          boats: {
-            data: [{
-              id: 2,
-              type: 'boat'
-            }]
-          }
         }
       }
     });
     adam = env.store.peekRecord('person', 1);
   });
 
-
   assert.equal(env.store.peekAll('person').get('length'), 1, 'one person record loaded');
   assert.equal(env.store._internalModelsFor('person').length, 1, 'one person internalModel loaded');
 
-  let relPayloads = env.store._relationshipsPayloads;
-
-  assert.equal(relPayloads.get('person', 1, 'cars').data.length, 1, 'one car relationship payload is cached');
-  assert.equal(relPayloads.get('person', 1, 'boats').data.length, 1, 'one boat relationship payload is cached');
-
-  run(function() {
```

Comment:
seems bad to remove so many asserts instead of changing them to not rely on _relationshipsPayloads ?